lib/python/rosie/vc.py

```diff
@@ -464,6 +606,47 @@ def create(argv):
             info_config = rose.config.load(info_file.name)
         finally:
             os.unlink(info_file.name)
+        new_id = ""
+        try:
+            info_config_new, report_error = client.create(info_config,
+                                                          from_id,
+                                                          opts.prefix,
+                                                          opts.meta_suite_mode,
+                                                          new_id, mode_cli=True)
+        except (RosePopenError, SuiteInfoFieldError,
+                SuiteIdOverflowError) as e:
+            report(e)
+            sys.exit(1)
+        while report_error is True:
+            info_file = tempfile.NamedTemporaryFile(delete=False)
+            info_config = info_config_new
+            if (info_config.get(["project"]).value is not None and
+                opts.project is None):
+                project = info_config.get(["project"]).value
+                info_config = client.generate_info_config(from_id,
+                                                          opts.prefix,
+                                                          project,
+                                                          info_config)
+            try:
+                rose.config.dump(info_config, info_file)
+                info_file.write(CREATE_INFO_CONFIG_COMMENT)
+                info_file.close()
```

Comment:
Should not need to close the file. Use `info_file.seek(0)` to reset location. `rose.config.load` will happily read a file handle. You should be able to get rid of the `delete=False` flag and the `try...finally` logic.